money/bostonhousing.qmd

```diff
@@ -0,0 +1,85 @@
+categories:
+  - linear regression
+  - EDA
+  - classification
+data:
+  year: 1978
+  files: HousingData.csv
+---
+
+## Motivation
+
+Harrison and Rubinfeld developed the dataset in 1978 to illustrate the issues with using housing market data to measure consumer willingness to pay for clean air. The authors use a hedonic pricing approach, which assumes that the price of a good or service can be modeled as a function of features both internal and external to the good or service. The input to this model was a dataset comprising the Boston Standard Metropolitan Statistical Area, with the nitric oxides concentration (NOX) serving as a proxy for air quality.
+
+This dataset was included as one of ``scikit-learn’s`` and tensorflow’s standard toy datasets. It has also been used as a benchmark tool for many machine learning papers. In 2020, users brought the dataset’s fairness issues to the scikit-learn development team (see scikit-learn issue [#16155](https://github.com/scikit-learn/scikit-learn/issues/16155)), after which the team decided to remove the dataset in scikit-learn version 1.2.
+
+Since then, fairness practitioners have used this dataset to demonstrate what fairness issues in machine learning could look like and how to assess them. This data project is inspired by the [Fairlearn project](https://fairlearn.org/v0.12/user_guide/datasets/boston_housing_data.html#references).
+
+## Data
+
+The data consists of 506 observations, collected from a mixture of surveys, administrative records, and other research papers. While the paper defines each variable and suggests their impacts on housing prices, it generally lacks reasoning for inclusion and attributes of variables. For example, not only it's problematic to include the percentage of Black residents as a predictive attributes, the variable `B` is parabolic with limited justifications.
+
+### Data preview
+
+```{r, echo=FALSE, results="asis"}
+source("../preview_dataset.R")
+preview_datasets()
+```
+
+### Variable descriptions
+
+| Variable | Description |
+|----|-------------|
+| CRIM | per capita crime rate by town |
+| ZN | proportion of residential land zoned for lots over 25,000 sq.ft. |
+| INDUS | proportion of non-retail business acres per town|
+| CHAS | Charles River dummy variable (= 1 if tract bounds river; 0 otherwise)|
+| NOX | 	nitric oxides concentration (parts per 10 million)|
+| RM | 	average number of rooms per dwelling |
+| AGE | proportion of owner-occupied units built prior to 1940|
+| DIS | weighted distances to five Boston employment centers |
+| RAD | index of accessibility to radial highways|
+| TAX | full-value property-tax rate per $10,000|
+| PTRATIO | pupil-teacher ratio by town |
+| B | 1000(Bk - 0.63)^2 where Bk is the proportion of Black people by town|
+| LSTAT | % lower status of the population, defined as proportion of adults without some high school education and proportion of male workers classified as laborers |

```

Comment:
Defined as two proportions -- is this proportion of people who meet either criterion, or some other combination of the two?